runner_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import unittest
+from app import app
+
+
+class LaunchingTest(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
```

Comment:
as you can see, it's *not* the app, but a client what you're getting. please change the attribute name to client